big-practice/src/assets/css/update-customer.css

```diff
@@ -0,0 +1,174 @@
+.modal-update {
+  width: 968px;
+  height: 797px;
+  position: absolute;
+  top: 290%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border: 1px solid black;
+  border-radius: 20px;
+  padding-top: 52px;
+  padding-left: 50px;
+  padding-right: 47px;
+}
+
+.modal-header {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.title {
+  color: var(--primary-color);
+  font-size: 36px;
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: var(--fw-semibold);
+  line-height: normal;
+  letter-spacing: 0.36px;
+  padding-bottom: 52px;
+}
+
+.sign-form {
+  display: grid;
+  grid-template-columns: auto auto;
+}
+
+.name-customer {
+  color: var(--accent-color);
+  font-size: 20px;
+  font-family: var(--secondary-font);
+  font-style: normal;
+  font-weight: var(--fw-bold);
+  line-height: 24px;
+  padding-bottom: 19px;
+}
+
+.fname {
+  width: 413px;
+  height: 62px;
+  border: 1px solid black;
```

Comment:
Should use hexadecimal color code for color variables. Please help check other places and update if any